controllers/workspace/status.go

```diff
@@ -320,3 +320,21 @@ func checkForFailingTimeout(workspace *common.DevWorkspaceWithConfig) (isTimedOu
 	}
 	return false, nil
 }
+
+func fmtStartTimeoutMessage(workspace *common.DevWorkspaceWithConfig) error {
+	status := workspaceConditionsFromClusterObject(workspace.Status.Conditions)
+	latestCondition := status.getFirstFalse()
+	if latestCondition != nil {
+		return fmt.Errorf("Devworkspace failed to progress past step '%s' for longer than timeout (%s)",
+			latestCondition.Message, workspace.Config.Workspace.ProgressTimeout)
+	}
+
+	latestTrueCondition := status.getLastTrue()
+	if latestTrueCondition != nil {
+		return fmt.Errorf("Devworkspace failed to progress past step '%s' for longer than timeout (%s)",
+			latestCondition.Message, workspace.Config.Workspace.ProgressTimeout)
```

Comment:
I think this was intended to be `latestTrueCondition.Message` instead of `latestCondition.Message`?

Reply:
Woopsie -- updated.